src/Geolocation/Geolocation.php

```diff
@@ -111,15 +115,20 @@ public function isUserGeolocated(string $formId, array $defaultLocations, array
 			return $formId;
 		}
 
+		$useLogger = $this->isCheckboxOptionChecked(SettingsTroubleshooting::SETTINGS_TROUBLESHOOTING_LOG_MODE_KEY, SettingsTroubleshooting::SETTINGS_TROUBLESHOOTING_DEBUGGING_KEY);
+
 		// Add ability to disable geolocation from external source. (Generaly used for GDPR).
 		$filterName = Filters::getGeolocationFilterName('disable');
 		if (\has_filter($filterName) && \apply_filters($filterName, null)) {
-			Helper::logger([
-				'geolocation' => 'Filter disabled active, skip geolocation.',
-				'formIdOriginal' => $formId,
-				'formIdUsed' => $formId,
-				'userLocation' => '',
-			]);
+			if ($useLogger) {
+				Helper::logger([
+					'geolocation' => 'Filter disabled active, skip geolocation.',
```

Comment:
```suggestion
					'geolocation' => 'Disable filter is active, skipping geolocation.',
```

This reads a bit better.